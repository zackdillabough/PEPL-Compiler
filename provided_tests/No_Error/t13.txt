POLY F1(X) = X^0 + X^1 + X^2 + X^3 + 1;
POLY F2 = x^2 + x + 1;
POLY F3 = x;
POLY F4 = x + 1;
POLY G1(P, Q, R, S, T) = P Q R S T;
POLY G2 = x + 2 x + 3 x;
START
INPUT X;
INPUT Y;
F4(X);
F1(Y);

INPUT X;
INPUT XYZ;
F3(XYZ);
F4(X);

INPUT Z;
INPUT XZ;
G1(Z, XZ, X, XYZ, Y);
G2(XZ);

34 21 4 34 21 4 34 21 4 34 21 4 34 21 4
