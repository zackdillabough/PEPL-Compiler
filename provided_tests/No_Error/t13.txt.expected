35
9725
34
5
239904
24
